inc/templates.php

```diff
@@ -363,14 +448,21 @@ function hydrate_components( array $components ) {
 		$component->provide_context( get_template_context() );
 
 		// Recursively hydrate children.
-		if ( ! empty( $component->get_children() ) ) {
+		if ( ! is_string( $component ) && ! empty( $component->get_children() ) ) {
```

Comment:
If we're converting a string to an `irving/text` component in `setup_component()` then I'm not sure when this would ever be a string.

Reply:
Remnant from an older PR from before we chatted.